src/components/FindOrganization/FindOrganization.js

```diff
@@ -14,97 +14,97 @@ import {
+    this.props.change('organization.name', o.name);
+    this.props.change('organization.id', o.id);
 
     this.setState(() => {
-      return { vendor: {
-        id: v.id,
-        name: v.name
+      return { organization: {
+        id: o.id,
+        name: o.name
       } };
     });
   }
 
-  updateVendorId = () => {
-    this.props.change('vendor.id', this.inputVendorId);
+  updateOrganizationId = () => {
+    this.props.change('organization.id', this.inputOrganizationId);
     this.setState(() => {
-      return { vendor: {
-        id: this.inputVendorId,
+      return { organization: {
+        id: this.inputOrganizationId,
         name: null
       } };
     });
   }
 
-  changeInputVendorId = (e) => {
-    this.inputVendorId = e.target.value;
+  changeInputOrganizationId = (e) => {
+    this.inputOrganizationId = e.target.value;
   }
 
-  renderVendorName = (vendor) => {
-    if (_.isEmpty(vendor.id)) {
+  renderOrganizationName = (organization) => {
+    if (_.isEmpty(organization.id)) {
       return null;
     }
 
-    const name = _.isEmpty(vendor.name) ?
-      <VendorName
-        vendorId={vendor.id}
+    const name = _.isEmpty(organization.name) ?
+      <OrganizationName
+        organizationId={organization.id}
         stripes={this.props.stripes}
       /> :
-      <div>{vendor.name}</div>;
+      <div>{organization.name}</div>;
 
     return (
       <div
-        name="vendorName"
+        name="organizationName"
         className={`${css.section} ${css.active}`}
       >
         <b>
-          {<FormattedMessage id="ui-finc-config.sourceManagement.vendor" />}
+          {<FormattedMessage id="ui-finc-config.sourceManagement.organization" />}
         </b>
         <div>{name}</div>
       </div>);
   }
 
   render() {
     const disableRecordCreation = true;
-    const vendorName = this.renderVendorName(this.state.vendor);
+    const organizationName = this.renderOrganizationName(this.state.organization);
 
-    const enterVendorIdButton =
+    const enterOrganizationIdButton =
       <Button
-        id="clickable-find-vendor-by-id"
-        onClick={this.updateVendorId}
+        id="clickable-find-organization-by-id"
+        onClick={this.updateOrganizationId}
       >
-        {<FormattedMessage id="ui-finc-config.findVendor.findVendorByIdButton" />}
+        {<FormattedMessage id="ui-finc-config.findOrganization.findOrganizationByIdButton" />}
       </Button>;
 
     const pluggable =
       <Pluggable
         aria-haspopup="true"
-        type="find-vendor"
-        id="clickable-find-vendor"
+        type="find-organization"
+        id="clickable-find-organization"
         {...this.props}
-        searchLabel="Vendor look-up"
+        searchLabel="Organization look-up"
         marginTop0
         searchButtonStyle="default"
-        dataKey="vendor"
-        selectVendor={this.selectVendor}
+        dataKey="organization"
+        selectOrganization={this.selectOrganization}
```

Comment:
I get an error when selecting an organization:
```
Uncaught TypeError: this.props.selectVendor is not a function
    at OrganizationSearchModal.passVendorOut (OrganizationSearchModal.js:43)
    at OrganizationSearchModal.passVendorOut (react-hot-loader.development.js:661)
    at Object.onSelectRow [as onRowClick] (SearchAndSort.js:439)
    at MCLRenderer.handleRowClick (MCLRenderer.js:443)
    at onClick (MCLRenderer.js:901)
    at HTMLUnknownElement.callCallback (react-dom.development.js:149)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:199)
    at invokeGuardedCallback (react-dom.development.js:256)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:270)
    at executeDispatch (react-dom.development.js:561)

```
ui-plugin-find-organization still uses the property 'selectVendor'. It was not renamed to 'selectOrganization' (https://github.com/folio-org/ui-plugin-find-organization/blob/master/OrganizationSearch/OrganizationSearchModal.js#L14).

```suggestion
        selectVendor={this.selectOrganization}
```